galaxy/templates/_helpers.tpl

```diff
@@ -167,25 +167,28 @@ Creates the bash command for the init container used to place files and change p
 cp -anL /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/mutable/integrated_tool_panel.xml;
 cp -anL /galaxy/server/config/sanitize_allowlist.txt /galaxy/server/config/mutable/sanitize_allowlist.txt;
 cp -anL /galaxy/server/config/data_manager_conf.xml.sample /galaxy/server/config/mutable/shed_data_manager_conf.xml;
-cp -anL /galaxy/server/config/tool_data_table_conf.xml.sample /galaxy/server/config/mutable/shed_tool_data_table_conf.xml;
 cp -aruL /galaxy/server/tool-data {{.Values.persistence.mountPath}}/;
 cp -aruL /galaxy/server/tools {{.Values.persistence.mountPath}}/;
+echo "<tables></tables>" > /galaxy/server/config/mutable/tool_data_table_conf.xml;
+echo "<tables></tables>" > /galaxy/server/config/mutable/shed_tool_data_table_conf.xml;
+chown 10001:10001 /galaxy/server/config/mutable/tool_data_table_conf.xml;
+chown 10001:10001 /galaxy/server/config/mutable/shed_tool_data_table_conf.xml;
```

Comment:
I think we can follow the same pattern as for `integrated_tool_panel.xml` and create a template file in galaxy/files/configs and place the sample shed_tool_data_table_conf.xml there? And then just do the usual `cp -anL`?